node/core/dispute-coordinator/src/participation/mod.rs

```diff
@@ -212,6 +212,19 @@ impl Participation {
 		Ok(())
 	}
 
+	/// Reprioritizes participation requests for disputes that are freshly included
+	pub async fn prioritize_newly_included<Context>(&mut self, ctx: &mut Context, included_receipts: &Vec<CandidateReceipt>) {
+		for receipt in included_receipts {
+			let r = self.queue.prioritize_if_present(ctx.sender(), receipt).await;
+			if let Err(queue_error) = r {
+				match queue_error {
+					QueueError::PriorityFull => return, // Avoid working through the rest of the vec
+					_ => (),
+				}
+			}
```

Comment:
```suggestion
			if let Err(QueueError::PriorityFull) =
				self.queue.prioritize_if_present(ctx.sender(), receipt).await
			{
				// Avoid working through the rest of the vec
				return
			}
```

Reply:
Why silently drop the error? Shouldn't we just propagate via ?